recipe/meta.yaml

```diff
@@ -1,20 +1,28 @@
 {% set name = "mamba" %}
-{% set libmamba_version = "1.5.11" %}
-{% set libmambapy_version = "1.5.11" %}
-{% set mamba_version = "1.5.11" %}
-{% set release = "2024.11.22" %}
+{% set libmamba_version = "2.0.5" %}
+{% set libmambapy_version = "2.0.5" %}
+{% set mamba_version = "2.0.5" %}
```

Comment:
Since there is a single source, I think it's safe to have a single version number, right?

Reply:
I agree with your comments, but we're trying to stay close to conda-forge